### PR TITLE
First version

### PR DESCRIPTION
**Pull request by fabienjuif**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  state = {
+    props: {},
+  }
+
+  getClassName = () => {
+    return `${this.props.className ? `${this.props.className} ` : ''}${className}`
+  }
+
+  animate = (animate, props) => {
```
Comment (frinyvonnick) [inline src/index.js]: param animate => shouldAnimate
function animate => applyClassName
param className => animatingClassName

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React, { Component, PropTypes } from 'react'
+import pick from 'lodash/pick'
+import isEqual from 'lodash/isEqual'
+
+const getDisplayName = (c) => c.displayName || c.name || 'Component'
+
+export default (
```
Comment (fabienjuif) [inline src/index.js]: Use destructuring instead of parameters.
README.md has to be updated.

**Comment by fabienjuif** [comment]
Add `"peerDependencies": {}`

**Comment by fabienjuif** [comment]
@frinyvonnick review for first merge plz

**Comment by frinyvonnick** [comment]
LGTM

**Pull request merged**